server/game/cards/08.6-SAT/ToGoForwardYouMustGoBack.js

```diff
@@ -0,0 +1,59 @@
+const _ = require('underscore');
+
+const DrawCard = require('../../drawcard.js');
+
+class ToGoForwardYouMustGoBack extends DrawCard {
+    setupCardAbilities() {
+        this.action({
```

Comment:
Needs to be limited to the dominance phase.

Reply:
(not shown here as outdated, but addressed in the PR)